Makefile

```diff
@@ -32,6 +32,10 @@ ifeq ($(CONFIG_SECURE_MODE),y)
 RUN_OPTIONS += -e SECURE=true
 endif
 
+ifeq ($CONFIG_CLOUD_SYNC),y)
+RUN_OPTIONS += -e CLOUD_SYNC=""
+endif
+                                                                                                              
```

Comment:
Where is the `CLOUD_SYNC` environment variable used?

Reply:
docker run -it -d --privileged --network="host" --name edge-orchestration -e **CLOUD_SYNC**="ec2-54-175-241-64.compute-1.amazonaws.com"  -v /var/edge-orchestration/:/var/edge-orchestration/:rw -v /var/run/docker.sock:/var/run/docker.sock:rw           -v /proc/:/process/:ro  lfedge/edge-home-orchestration-go:latest

Reply:
used to set to url of broker if cloudsync to be used else kept nil. this is used till the api is exposed for testing

Reply:
this environment varisable is used in controller to see if cloudsync has to be started or not